tools/reg_parse.c

```diff
@@ -1058,7 +1058,7 @@ check_dimspecs()
 		  p->assoc_nl_var_s,p->name ) ;
 	  return(1) ;
         }
-        if ( ! q->node_kind & RCONFIG )
+        if ( (! q->node_kind) & RCONFIG )
```

Comment:
```suggestion
        if ( ! (q->node_kind & RCONFIG) )
```
or
```suggestion
        if ( (~ q->node_kind) & RCONFIG )
```
I suspect `RCONFIG != 1`, so this conditional is unlikely to be met as-is.